plugins/gzip/sample.gzip.config

```diff
@@ -29,7 +29,10 @@
 #
 # compressible-content-type: wildcard pattern for matching compressible content types
 #
-# disallow: wildcard pattern for disablign compression on urls
+# disallow: wildcard pattern for disabling compression on urls
+#
+# minimum-content-length: minimum content length for compression to be enabled (in bytes)
```

Comment:
Should we point out here that this only works if the content has a CL: header? I.e. chunked encoding would not be possible to to this check for I think, since you would have to check the body for the length.